themes/shezaan/shezaan.theme.bash

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+SCM_THEME_PROMPT_DIRTY=" ${red}✗"
+SCM_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+SCM_THEME_PROMPT_PREFIX=" |"
+SCM_THEME_PROMPT_SUFFIX="${green}|"
+
+GIT_THEME_PROMPT_DIRTY=" ${red}✗"
+GIT_THEME_PROMPT_CLEAN=" ${bold_green}✓"
+GIT_THEME_PROMPT_PREFIX=" ${green}|"
+GIT_THEME_PROMPT_SUFFIX="${green}|"
+
+RVM_THEME_PROMPT_PREFIX="|"
+RVM_THEME_PROMPT_SUFFIX="|"
+
+myip=`$BASH_IT/lib/get_ip.py`
```

Comment:
You're welcome!

As a first step, you could inline the function in the theme, like `python -c "import socket; print(socket.gethostbyname(socket.gethostname()));"` - this would avoid the additional file in `lib`.

If you could make that change, I'll gladly merge your theme. If you or someone else then wants to make this functionality available to other themes, we can make the necessary changes later.